Host a Static Website with Hugo and NGINX.md

```diff
@@ -155,7 +154,7 @@ Use the `wget` command to get the latest Hugo release files. Latest releases are
     wget https://github.com/gohugoio/hugo/releases/download/v0.31.1/hugo_0.31.1_Linux-64bit.deb
 
 {{< note >}}
-To get the link to the release you wish to obtain right click on the release link and select Copy Link. Paste in the Terminal window.
+To get the link to the release you wish to obtain right click on the release link and select "Copy Link". Paste in the Terminal window.
```

Comment:
Ah, "To get the link of the release..."

Reply:
I saw that comment. Issue is I had a hard time finding the link to the release even with the releases listed. If you click on a release it automatically downloads. Now you are stuck with deciding if using FTP is a better option since it's downloaded already. Even copy + paste into most applications hides the actual URL by making it a clickable link. So getting the URL may not be as obvious as it appears.